pkg/bootstrap/config.go

```diff
@@ -49,6 +49,7 @@ const (
 	// required stats are used by readiness checks.
 	requiredEnvoyStatsMatcherInclusionPrefixes = "cluster_manager,listener_manager,http_mixer_filter,tcp_mixer_filter,server,cluster.xds-grpc"
 	requiredEnvoyStatsMatcherInclusionSuffix   = "ssl_context_update_by_sds"
+	requiredEnvoyStatsMatcherInclusionRegexps  = "cluster.*.outlier_detection.*"
```

Comment:
We need to measure the performance impact of this addition.
Prefix and suffix matches are much cheaper than regex. Making a regex part of standard must not cause too much perf loss.

Can you paste some examples of what these stats are?

Reply:
Can you paste some examples of what these stats are ?

these metrics are generated between any two svc. e.g. 
```
cluster.outbound|80||ubuntu-service.istio-test.svc.cluster.local.outlier_detection.ejections_active: 0
cluster.outbound|80||ubuntu-service.istio-test.svc.cluster.local.outlier_detection.ejections_consecutive_5xx: 0
cluster.outbound|80||ubuntu-service.istio-test.svc.cluster.local.outlier_detection.ejections_detected_consecutive_5xx: 0
cluster.outbound|80||ubuntu-service.istio-test.svc.cluster.local.outlier_detection.ejections_detected_consecutive_gateway_failure: 0
```

these metrics can be found here:
https://www.envoyproxy.io/docs/envoy/latest/configuration/upstream/cluster_manager/cluster_stats#outlier-detection-statistics

the format of these metrics are `cluster.<name>.outlier_detection.` thats why we needs `regex` here.

Reply:
If you can turn this into a set of suffixes it would be better. If the above list is exhaustive then enumerating ejection_-* would do the job.

Reply:
> enumerating ejection_-* would do the job

it's possible but too many: there are about 15~20 metrics.

if you insisted on `suffix` maybe I can figure out a subset of these metrics, about 5 - 10 item, does 5 - 10 suffix ok ? @mandarjog

Reply:
5 would be good. 
The alternative is to add doc to update annotations in the injector template.

Reply:
> 5 would be good

Cool, will have a try.

Reply:
DONE, PTAL. @mandarjog